core/user.go

```diff
@@ -94,7 +94,7 @@ func (u *User) Validate() error {
 	switch {
 	case !govalidator.IsByteLength(u.Login, 1, 50):
 		return errUsernameLen
-	case !govalidator.Matches(u.Login, "^[a-zA-Z0-9_-]+$"):
+	case !govalidator.Matches(u.Login, "^[.a-zA-Z0-9_-]+$"):
```

Comment:
Not sure about that, not should the dot be escaped?

Reply:
It doesn't need to be escaped inside brackets. also i add a test for it and it didn't break any previous cases.